src/asciidoc/docs/develop/publishLocal.adoc

```diff
@@ -0,0 +1,240 @@
+= Publishing a project you developed locally
+:page-slug: publish-locally-developed
+:page-description: How to push local changes to the origin repository and pull them into the Koji editor, so you can publish the project.
+
+In the <<work-locally#, previous section>> of this tutorial, you learned how to use Git to clone a Koji project to your development machine and how to install and launch a Node.js server to run your project locally.
+
+In this section, you will
+// tag::description[]
+learn how to push the changes that you make locally to your project's origin repository, and how to pull the latest version of your project into the Koji editor, where you can publish it to the world.
```

Comment:
Since you no longer need to publish from the editor, we can simplify this intro. E.g.
```suggestion
learn how to push the changes that you make locally to your project's origin repository, so you can publish it to the world.
```